### PR TITLE
Tests runner

### PR DESCRIPTION
**Pull request by GirZ0n**

### CODE REVIEW

File: jba/src/jba/tests/tests_parser.py [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+                ),
+            )
+
+    if not exceptions:
+        logger.info('There are no exceptions in the stderr logs file')
+
+    return exceptions
+
+
+def _parse_test_logs(submission_gradle_logs_path: Path) -> Optional[List[TestData]]:
```
Comment (nbirillo) [inline jba/src/jba/tests/tests_parser.py]: I'w prefer to separate this function into several ones, also maybe it is better to move HTML parsing into another file

File: jba/src/jba/tests/tests_parser.py [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+from jba.models.edu_columns import EduColumnName
+from jba.tests.tests_runner import GRADLE_STDERR_LOGS_FILE, TEST_LOGS_FOLDER_NAME
+
+logger = logging.getLogger(__name__)
+
+EXCEPTION_REGEXP = re.compile(r'^e: (.*): \((\d+), (\d+)\): (.*)$')
+
+
+@unique
+class TestDataField(Enum):
```
Comment (nbirillo) [inline jba/src/jba/tests/tests_parser.py]: Could you please move models classes into Models file in this package?

**Review by nbirillo — CHANGES_REQUESTED** [review]
It looks awesome, I have only small comments. Also, you need to add tests - for the functionality, another one that just runs the scripts

**Comment by GirZ0n** [comment]
Closed due to splitting into #18 and #19

**Pull request closed without merging**